Userland/Services/WebDriver/Client.cpp

```diff
@@ -442,6 +443,23 @@ ErrorOr<JsonValue, HttpError> Client::handle_get_timeouts(Vector<AK::StringView>
     return make_json_value(timeouts);
 }
 
+// 9.2 Set Timeouts, https://w3c.github.io/webdriver/#dfn-set-timeouts
+// POST /session/{session id}/timeouts
+ErrorOr<JsonValue, HttpError> Client::handle_set_timeouts(Vector<AK::StringView> const& parameters, AK::JsonValue const& payload)
+{
+    dbgln_if(WEBDRIVER_DEBUG, "Handling POST /session/<session id>/timeouts");
+    auto* session = TRY(find_session_with_id(parameters[0]));
+
+    // 1. Let timeouts be the result of trying to JSON deserialize as a timeouts configuration the request’s parameters.
+    auto timeouts = TRY(json_deserialize_as_a_timeouts_configuration(payload));
+
+    // 2. Make the session timeouts the new timeouts.
+    session->set_timeouts_configuration(timeouts);
+
+    // 3. Return success with data null.
+    return make_json_value(JsonValue {});
```

Comment:
I mostly see `JsonValue()`, is there a reason to use `{}` here? whatever we do, it should be consistent...

Reply:
https://stackoverflow.com/a/18222927/5952681 - no _reason_, but I'd prefer `{}` generally.